yin_yang/plugins/icons.py

```diff
@@ -64,3 +67,35 @@ def available_themes(self) -> dict:
                 themes.extend(d.name for d in entries if d.is_dir() and path.isfile(d.path + '/index.theme'))
 
         return {t: t for t in themes}
+
+class _Kde(PluginCommandline):
+    def __init__(self):
+        super().__init__(["/usr/lib/plasma-changeicons", r"{theme}"])
+        self.theme_light = "breeze"
+        self.theme_dark = "breeze-dark"
+
+    @property
+    def available_themes(self) -> dict:
+        themes = []
+
+        for icon_theme in theme_directories:
+            if not path.isdir(icon_theme):
+                continue
+
+            for icon_theme_folder in scandir(icon_theme):
+                if not all(
+                    (
+                        icon_theme_folder.is_dir(),
+                        path.isfile(icon_theme_folder.path + "/index.theme"),
```

Comment:
If I remember correctly, you can do `(icon_theme_folder / 'index.theme').is_file()`. I think that looks a bit nicer.

Reply:
Seems not:
![Screenshot_20240403_212745](https://github.com/oskarsh/Yin-Yang/assets/1223421/98765c7c-e505-4711-be64-98e7b56cc131)

Reply:
We are using `scandir` from `os`, not `pathlib`